easybuild/tools/systemtools.py

```diff
@@ -406,7 +406,11 @@ def get_cpu_features():
                 # for Linux@POWER systems, no flags/features are listed, but we can check for Altivec
                 cpu_altivec_regex = re.compile("^cpu\s*:.*altivec supported", re.M)
                 if cpu_altivec_regex.search(proc_cpuinfo):
-                    cpu_feat = ['altivec']
+                    cpu_feat.append('altivec')
+                # VSX is supported since POWER7
+                cpu_power7_regex = re.compile("^cpu\s*:.*POWER(7|8|9)", re.M)
```

Comment:
I liked it.... made sense to me (for now)...  The following (which seems to build nicely though I haven't tested it., uses the same logic:

http://siliconslick.com/easybuild/easyconfigs/curie/curie/FFTW/FFTW-3.3.5-gompi-2016D.eb